pkg/tagger/remote/grpclog.go

```diff
@@ -0,0 +1,45 @@
+package remote
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"google.golang.org/grpc/grpclog"
+)
+
+type logLevel int
+
+const (
+	logLevelInfo logLevel = iota
+	logLevelWarn
+	logLevelErr
+)
+
+type redirectLogger struct {
+	level logLevel
+}
+
+func (l redirectLogger) Write(b []byte) (int, error) {
+	msg := string(b)
+
+	switch l.level {
+	case logLevelInfo:
+		log.InfoStackDepth(stackDepth, msg)
+	case logLevelWarn:
+		log.WarnStackDepth(stackDepth, msg)
+	case logLevelErr:
+		log.ErrorStackDepth(stackDepth, msg)
+	default:
+		log.InfoStackDepth(stackDepth, msg)
+	}
+
+	return 0, nil
+}
+
+const stackDepth = 7
+
+func init() {
+	grpclog.SetLoggerV2(grpclog.NewLoggerV2(
+		redirectLogger{logLevelInfo},
+		redirectLogger{logLevelWarn},
+		redirectLogger{logLevelErr},
```

Comment:
nits to be futur proof.

```suggestion
		redirectLogger{level: logLevelInfo},
		redirectLogger{level: logLevelWarn},
		redirectLogger{level: logLevelErr},
```